lib/init.coffee

```diff
@@ -162,10 +170,14 @@ module.exports =
     ## We want column number so that we can know where to underline.
     params.push("--show-column-numbers")
 
-    ## We only want to report warnings about the requested file and not about its dependencies
-    ## Note: This silencing of the warnings external to the current file is not perfect, it will still report warnings about *.pyi files which will need to be filtered later.
-    params.push("--follow-imports")
-    params.push("silent")
+    if (@followImports)
+      params.push("--follow-imports")
+      params.push(@followImports)
+    else
```

Comment:
Since the setting has a default value, I am pretty sure this else condition is unreachable... could you validate... if it is effectively unreachable than the whole "if else" can be remove and only keep those two lines:

>     params.push("--follow-imports")
>     params.push(@followImports)